cmd/ftl/cmd_dev.go

```diff
@@ -79,6 +79,22 @@ func (d *devCmd) Run(
 	}
 	os.Setenv("FTL_DEV_DIRS", strings.Join(absDirs, ","))
 
+	// Validate module directories before launching console or starting build engine.
```

Comment:
I don't think this is needed. These cases are already be handled in `main`, though it was broken as of last week (fixed here: https://github.com/block/ftl/pull/5180):
```
ftl: error: [<dirs> ...]: stat /Users/mtoohey/Code/ftl/examples/javafake: no such file or directory
🐚 mtoohey ~/Code/ftl [main] $ ftl dev fakedir          
Usage: ftl dev [<dirs> ...] [flags]
Run "ftl dev --help" for more information.
```
```
ftl: error: [<dirs> ...]: stat /Users/mtoohey/Code/ftl/fakedir: no such file or directory
🐚 mtoohey ~/Code/ftl [main] $ ftl dev Dockerfile
Usage: ftl dev [<dirs> ...] [flags]
Run "ftl dev --help" for more information.

ftl: error: [<dirs> ...]: "/Users/mtoohey/Code/ftl/Dockerfile" exists but is not a directory
```

Reply:
Back when I create the issue about this was when I fixed it made that PR, but the issue remained because the interactive console code was eating the error for some reason. But that appears to be fixed now... 🤷‍♂️